sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -322,9 +322,14 @@ object ScalaJSPluginInternal {
 
             IO.createDirectory(output.getParentFile)
 
-            linker.link(ir, moduleInitializers,
+            try {
+              linker.link(ir, moduleInitializers,
                 AtomicWritableFileVirtualJSFile(output),
                 sbtLogger2ToolsLogger(log))
```

Comment:
These two lines must also be indented two more spaces now.

Reply:
Added indent and mention LinkingException in a squashed commit.